package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "testswitch-frontend-candidate",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/techswitch-learners/testswitch-frontend-candidate.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/techswitch-learners/testswitch-frontend-candidate/issues"
+  },
+  "homepage": "https://github.com/techswitch-learners/testswitch-frontend-candidate#readme",
+  "dependencies": {
+    "@zeit/next-css": "^1.0.1",
+    "@zeit/next-sass": "^1.0.1",
+    "next": "^9.3.4",
+    "node-sass": "^4.13.1",
```

Comment:
And I _think_ `@zeit/next-sass` and `node-sass` can both just be dev-dependencies.
This should help keep the size of the deployed application nice and small! :)

Reply:
have moved to dev dependancies